Jenkinsfile

```
@@ -374,53 +375,17 @@ lock(resource: "build-${params.STREAM}") {
                 }
             }
 
-            stage('Build Azure') {
-                utils.shwrap("""
-                cosa buildextend-azure
-                """)
-            }
-
-            stage('Build Exoscale') {
-                utils.shwrap("""
-                cosa buildextend-exoscale
-                """)
-            }
-
-            stage('Build Openstack') {
-                utils.shwrap("""
-                cosa buildextend-openstack
-                """)
-            }
-
-            stage('Build Aliyun') {
-                utils.shwrap("""
-                cosa buildextend-aliyun
-                """)
-            }
-
-            stage('Build Vultr') {
-                utils.shwrap("""
-                cosa buildextend-vultr
-                """)
-            }
-
-            stage('Build VMware') {
-                utils.shwrap("""
-                cosa buildextend-vmware
-                """)
-            }
-
-            stage('Build GCP') {
-                utils.shwrap("""
-                cosa buildextend-gcp
-                """)
-            }
-
-            stage('Build DigitalOcean') {
-                utils.shwrap("""
-                cosa buildextend-digitalocean
-                """)
+            // parallel build these artifacts
+            pbuilds = [:]
+            ["Azure", "Exoscale", "OpenShift", "Aliyun", "Vultr", "Vware", "GCP", "DigitalOcean"].each {
+                pbuilds[it] = {
+                    cmd = it.toLowerCase()
```

```suggestion
                    def cmd = it.toLowerCase()
```

Otherwise the variable outlives the scope.